documentation/src/test/resources/META-INF/persistence.xml

```diff
@@ -54,35 +43,6 @@
             <property name="hibernate.search.backend.password"
                       value="j@rV1s"/>
             <!-- end::gettingstarted-configuration-orm_elasticsearch[] -->
-
-            <!--
-                Not to be included in the documentation:
-                this configuration is only relevant in our own testing infrastructure.
-                WARNING: if you update the property names below,
-                take care to also update the name of the commented properties just above.
-             -->
-            <property name="hibernate.search.schema_management.strategy"
-                      value="drop-and-create-and-drop"/>
-            <property name="hibernate.search.backend.hosts"
-                      value="${test.elasticsearch.connection.hosts}"/>
-            <property name="hibernate.search.backend.protocol"
-                      value="${test.elasticsearch.connection.protocol}"/>
-            <property name="hibernate.search.backend.username"
-                      value="${test.elasticsearch.connection.username}"/>
-            <property name="hibernate.search.backend.password"
-                      value="${test.elasticsearch.connection.password}"/>
-            <property name="hibernate.search.backend.aws.signing.enabled"
-                      value="${test.elasticsearch.connection.aws.signing.enabled}"/>
-            <property name="hibernate.search.backend.aws.region"
-                      value="${test.elasticsearch.connection.aws.region}"/>
-            <property name="hibernate.search.backend.aws.credentials.type"
-                      value="${test.elasticsearch.connection.aws.credentials.type}"/>
-            <property name="hibernate.search.backend.aws.credentials.access_key_id"
-                      value="${test.elasticsearch.connection.aws.credentials.access_key_id}"/>
-            <property name="hibernate.search.backend.aws.credentials.secret_access_key"
-                      value="${test.elasticsearch.connection.aws.credentials.secret_access_key}"/>
-            <property name="hibernate.search.automatic_indexing.synchronization.strategy"
-                      value="sync"/>
```

Comment:
Removing the update of these properties can be an issue for the correct execution of `GettingStartedDefaultAnalysisIT` test.

Reply:
I think that the issue is related to the fact that host+port takes the precedence over the uris...

Reply:
Yes, I'll look into it.